test/test_grid_dofhandler_vtk.jl

```diff
@@ -321,36 +328,38 @@ end
+    @test topology.vertex_vertex_neighbor[5,2] == [VertexIndex(4,4)]
+    @test topology.vertex_vertex_neighbor[6,1] == [VertexIndex(3,3)]
     @test isempty(getneighborhood(topology, quadgrid, VertexIndex(2,2)))
     @test length(getneighborhood(topology, quadgrid, VertexIndex(2,4))) == 3
     #test face neighbor maps cellid and local face id to neighbor id and neighbor local face id
-    @test topology.edge_edge_neighbor[1,2] == Ferrite.EntityNeighborhood(EdgeIndex(2,4))
-    @test topology.edge_edge_neighbor[1,3] == Ferrite.EntityNeighborhood(EdgeIndex(3,1))
-    @test topology.edge_edge_neighbor[2,3] == Ferrite.EntityNeighborhood(EdgeIndex(4,1))
-    @test topology.edge_edge_neighbor[2,4] == Ferrite.EntityNeighborhood(EdgeIndex(1,2))
-    @test topology.edge_edge_neighbor[3,1] == Ferrite.EntityNeighborhood(EdgeIndex(1,3))
-    @test topology.edge_edge_neighbor[3,2] == Ferrite.EntityNeighborhood(EdgeIndex(4,4))
-    @test topology.edge_edge_neighbor[3,3] == Ferrite.EntityNeighborhood(EdgeIndex(5,1))
-    @test topology.edge_edge_neighbor[4,1] == Ferrite.EntityNeighborhood(EdgeIndex(2,3))
-    @test topology.edge_edge_neighbor[4,3] == Ferrite.EntityNeighborhood(EdgeIndex(6,1))
-    @test topology.edge_edge_neighbor[4,4] == Ferrite.EntityNeighborhood(EdgeIndex(3,2))
-    @test topology.edge_edge_neighbor[5,1] == Ferrite.EntityNeighborhood(EdgeIndex(3,3))
-    @test topology.edge_edge_neighbor[5,2] == Ferrite.EntityNeighborhood(EdgeIndex(6,4))
-    @test topology.edge_edge_neighbor[5,3] == Ferrite.EntityNeighborhood(Ferrite.BoundaryIndex[])
-    @test topology.edge_edge_neighbor[5,4] == Ferrite.EntityNeighborhood(Ferrite.BoundaryIndex[])
-    @test topology.edge_edge_neighbor[6,1] == Ferrite.EntityNeighborhood(EdgeIndex(4,3))
-    @test topology.edge_edge_neighbor[6,2] == Ferrite.EntityNeighborhood(Ferrite.BoundaryIndex[])
-    @test topology.edge_edge_neighbor[6,3] == Ferrite.EntityNeighborhood(Ferrite.BoundaryIndex[])
-    @test topology.edge_edge_neighbor[6,4] == Ferrite.EntityNeighborhood(EdgeIndex(5,2))
+    @test topology.edge_edge_neighbor[1,2] == [EdgeIndex(2,4)]
+    @test topology.edge_edge_neighbor[1,3] == [EdgeIndex(3,1)]
+    @test topology.edge_edge_neighbor[2,3] == [EdgeIndex(4,1)]
+    @test topology.edge_edge_neighbor[2,4] == [EdgeIndex(1,2)]
+    @test topology.edge_edge_neighbor[3,1] == [EdgeIndex(1,3)]
+    @test topology.edge_edge_neighbor[3,2] == [EdgeIndex(4,4)]
+    @test topology.edge_edge_neighbor[3,3] == [EdgeIndex(5,1)]
+    @test topology.edge_edge_neighbor[4,1] == [EdgeIndex(2,3)]
+    @test topology.edge_edge_neighbor[4,3] == [EdgeIndex(6,1)]
+    @test topology.edge_edge_neighbor[4,4] == [EdgeIndex(3,2)]
+    @test topology.edge_edge_neighbor[5,1] == [EdgeIndex(3,3)]
+    @test topology.edge_edge_neighbor[5,2] == [EdgeIndex(6,4)]
+    @test topology.edge_edge_neighbor[5,3] == Ferrite.BoundaryIndex[]
+    @test topology.edge_edge_neighbor[5,4] == Ferrite.BoundaryIndex[]
```

Comment:
So the type instability is not fixed here?
```suggestion
    @test topology.edge_edge_neighbor[5,3] == Ferrite.EdgeIndex[]
    @test topology.edge_edge_neighbor[5,4] == Ferrite.EdgeIndex[]
```

(also below)

Reply:
I was not aware of this being due to type-instabilities, there should be no such for this test for sure! 
But this is indeed a strange test at least now, since e.g. 
```julia
julia> String[] == Int[]
true
```
I guess we should rather test
```suggestion
    @test isempty(topology.edge_edge_neighbor[5,3])
    @test isempty(topology.edge_edge_neighbor[5,4])
```

Reply:
Right